katas/args/jhon/parser.js

```diff
@@ -0,0 +1,101 @@
+export class Parser {
+    constructor() {
+        this.args = process.argv[2];
+        this.schema = {
+            param0 :{
+                id: 0,
+                symbol: 'l',
+                type: 'boolean'
+            },
+            param1 :{
+                id: 1,
+                symbol: 'p',
+                type: 'int'
+            },
+            param2 :{
+                id: '2',
+                symbol: 'd',
+                type: 'String'
+            }
+        };
+    }
+
+    parser(args) {
```

Comment:
this function is good, but is getting too much code, you might consider creating functions that divide the logic